sormas-ui/src/main/java/de/symeda/sormas/ui/user/UserEditForm.java

```diff
@@ -229,7 +248,7 @@ private void updateFieldsByUserRole() {
 		}
 
 		final ComboBox healthFacility = (ComboBox) getFieldGroup().getField(UserDto.HEALTH_FACILITY);
-		healthFacility.setVisible(hasOptionalHealthFacility || useHealthFacility);
+		healthFacility.setVisible(useHealthFacility || hasOptionalHealthFacility);
```

Comment:
Would probably be better to set useHealthFacility to true in line 230 when hasOptionalHealthFacility is true. Thisway  useDistrict and useRegion automatically have the right value. I think with the current solution the region field would be missing.

Reply:
useHealthFacility is needed on the line below to set the facility field required.

I will change the visibility of the region field to `region.setVisible(useRegion || hasOptionalHealthFacility);`

Reply:
Ah, I see. Ok